framework/widget/src/main/java/org/apache/ofbiz/widget/renderer/macro/RenderableFtlFormElementsBuilder.java

```diff
@@ -287,7 +287,7 @@ public RenderableFtl textField(final Map<String, Object> context, final ModelFor
                 .stringParameter("id", id)
                 .stringParameter("event", event != null ? event : "")
                 .stringParameter("action", action != null ? action : "")
-                .booleanParameter("disabled", disabled)
+                .stringParameter("disabled", String.valueOf(disabled))
```

Comment:
I don't think this change is needed. FTL is able to accept boolean parameters. 
One of the ideas behind RenderableFtl's parameters was to avoid encoding meaning in strings when other more focussed types, such as boolean an int, would do a better job.

Reply:
I'll switch to a `boolean` in `HtmlFormMacroLibrary.ftl`, it's a little bit cleaner indeed, so this change is no longer needed (vs comparing a `string` to `true` in the macros).